arangod/Aql/AqlItemBlockManager.h

```diff
@@ -110,6 +116,8 @@ class AqlItemBlockManager {
   };
 
   Bucket _buckets[numBuckets];
+
+  AqlItemBlock* _constValueBlock = nullptr;
```

Comment:
Can we make this an `std::unique_ptr<AqlItemBlock>` instead of a raw pointer?
```suggestion
  AqlItemBlock* _constValueBlock = nullptr;
```

Reply:
Unfortunately not, because the `AqlItemBlock` destructor is protected.